docs/src/interfacer.md

```diff
@@ -76,8 +76,34 @@ functions must be extended for all models being run.
 
 | Coupler name      | Description | Units | Default value |
 |-------------------|-------------|-------|---------------|
-| `energy` | globally integrated energy | J | `nothing` |
```

Comment:
This hasn't changed in the code, but this description was wrong. We get energy/water per surface area from each component, then within the coupler we integrate it horizontally